js_tests/qunit/qunit.js

```diff
@@ -3193,7 +3193,7 @@ QUnit.diff = ( function() {
 				// Upon reaching an equality, check for prior redundancies.
```

Comment:
We are vendoring quint, so we shouldn't make changes to it but rather contribute the fixes upstream (if you want).